src/EventStore.ClientAPI/Messages/ClusterMessages.cs

```diff
@@ -65,24 +65,26 @@ public class MemberInfoDto {
 			}
 		}
 
+		//The order specified in this enum is important.
+		//It defines the default sorting order (descending, ignoring a few specific states) used in ClusterDnsEndPointDiscoverer when choosing which node the client should connect to.
 		public enum VNodeState {
-			Initializing,
-			Unknown,
-			PreReplica,
-			CatchingUp,
-			Clone,
-			Slave,
-			Follower,
-			PreMaster,
-			PreLeader,
-			Master,
-			Leader,
-			Manager,
-			ShuttingDown,
-			Shutdown,
-			ReadOnlyLeaderless,
-			PreReadOnlyReplica,
-			ReadOnlyReplica
+			Initializing = 1,
```

Comment:
It would be good to put Unknown back in at 0 as that is the default value for the enum

Reply:
If we set `Unknown` to 0, it will also imply that clients will favor nodes in `Initializing` and `ReadOnlyLeaderLess` states over nodes in the `Unknown` state (both `Unknown` and `ReadOnlyLeaderLess` accept reads). Is it ok or should we set `Initializing` to 0 instead?